src/components/04_templates/ContentTable/ContentTable.js

```diff
@@ -194,7 +201,7 @@ class ContentTable extends React.Component {
                           to={`/user/${
                             this.props.includes['user--user'][
                               relationships.uid.data.id
-                            ].attributes.uid
```

Comment:
Once https://www.drupal.org/project/drupal/issues/2353611 lands, you won't have to do this anymore, you'll be able to generate URLs like `/user/f7649197-4133-4ee7-af6a-67ea32b32a77` :)

Reply:
@wimleers Except that we would not want to do that. We're matching how Drupal handles urls right now, so we can slowly take over routes, like `/user/5`

Reply:
Makes sense. Well, that's why it's prefixed with `drupal_internal__`: to signal to JSON API clients that you'd only ever want to use this if you're building something tightly coupled. Like … an editing UI that integrates with the existing Drupal UI :)